avro-serializer/src/main/java/io/confluent/kafka/serializers/AvroSchemaUtils.java

```diff
@@ -31,6 +31,7 @@
 
   static {
     Schema.Parser parser = new Schema.Parser();
+    parser.setValidateDefaults(false);
```

Comment:
Is it not possible to set it to false inside Schema.Parser() ? I'm assuming it needs to be set everywhere that function call is done?

Reply:
It's not possible to set while calling the constructor.  The calls in this PR are the ones where it needs to be set.

Reply:
Ok. LGTM then.